3.7/graphs-satellite-graphs-details.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: default
+description: This chapter describes the usage of SatelliteGraphs in Detail
+title: SatelliteGraphs In Detail
+---
+Satellite Graph Details
+======================
+
+This chapter explains the details of the usage of a SatelliteGraphs, 
+including more advanced configuration possibilities.
+
+The Initial Collection
+--------------
+
+To create a SatelliteGraph manually managed we'll explain a little bit more
```

Comment:
this sentence does not make sense to me,
I think the `managed` should be removed or?